fastavro/reader.py

```diff
@@ -134,6 +142,87 @@ def read_boolean(fo, writer_schema=None, reader_schema=None):
     return unpack('B', fo.read(1))[0] != 0
 
 
+def read_timestamp_millis(data, writer_schema=None, reader_schema=None):
```

Comment:
Combine this one with `read_timestamp_micros`, add a parameter for the resolution.